cli/popper/utils.py

```diff
@@ -266,3 +272,41 @@ def fetch_readme_for_repo(user, repo, path_to_action, version):
                        user, repo, version, path_to_action, 'README.md')
     r = make_gh_request(url, err=False)
     return r.text
+
+
+def convert_to_singularityfile(dockerfile, singularityfile):
```

Comment:
since these utility functions are only used by the `SingularityRunner` class, can we move them to be static methods of it?